accounts/abi/bind/base.go

```diff
@@ -424,6 +430,25 @@ func (c *BoundContract) createPrivateTransaction(tx *types.Transaction, payload
 	return privateTx
 }
 
+// (Quorum) createMarkerTx creates a new public privacy marker transaction for the given private tx, distributing tx to the specified privateFor recipients
+func (c *BoundContract) createMarkerTx(opts *TransactOpts, tx *types.Transaction, args PrivateTxArgs) (*types.Transaction, error) {
+	// Choose signer to sign transaction
+	if opts.Signer == nil {
+		return nil, errors.New("no signer to authorize the transaction with")
+	}
+	signedTx, err := opts.Signer(types.QuorumPrivateTxSigner{}, opts.From, tx)
+	if err != nil {
+		return nil, err
+	}
+
+	hash, err := c.transactor.DistributeTransaction(ensureContext(opts.Context), signedTx, args)
+	if err != nil {
+		return nil, err
+	}
+
+	return types.NewTransaction(signedTx.Nonce(), common.QuorumPrivacyPrecompileContractAddress(), tx.Value(), tx.Gas(), tx.GasPrice(), common.FromHex(hash)), nil
```

Comment:
shouldn't be the gas limit here the same as https://github.com/ConsenSys/quorum/pull/1234/files#diff-c426ecd2f7d247753b9ea8c1cc003f21fa412661c1f967d203d4edf8163da344R2884?

Reply:
Good point, ideally it would also use the Intrinsic Gas value here too. I'll make that change.
It'll take a day or so, as I'll need to run acceptance tests against it to ensure it doesn't cause nay other issues. If necessary, it can be added after the PR is merged (to avoid holding up other merges to master).

Reply:
no need to add after, as we need to have this ready for release so lets put all in one PR that we can merge and then ready for release 👍 (other PRs won't go in in the same release)

Reply:
Ok, I've just committed a change to do use IntrinsicGas for the abi generated PMT.